src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -132,6 +132,7 @@
     r'cannot open shared object file')
 LINUX_GDB_CRASH_TYPE_REGEX = re.compile(r'Program received signal ([a-zA-Z]+),')
 LINUX_GDB_CRASH_ADDRESS_REGEX = re.compile(r'rip[ ]+([xX0-9a-fA-F]+)')
+LINUX_GDB_CRASH_ADDRESS_NO_REGISTERS_REGEX = re.compile(r'^([xX0-9a-fA-F]+)\s+in\s+')
```

Comment:
Since this regex is too generic, like it can match "abc in def", can you change to like
re.compile(r'^0x([0-9a-fA-F]+)\s+in\s+')